### PR TITLE
docs: fix simple typo, clearning -> cleaning

### PR DESCRIPTION
**Pull request by timgates42**
There is a small typo in CONTRIBUTING.md.

Should read `cleaning` rather than `clearning`.

Semi-automated pull request generated by
https://github.com/timgates42/meticulous/blob/master/docs/NOTE.md

### CODE REVIEW

**Review by sandrobonazzola — APPROVED** [review]
LGTM

**Comment by sandrobonazzola** [comment]
@mingrammer can you please review/merge?

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -55,7 +55,7 @@ or update the `ALIASES` map in [config.py](config.py).
 Then just run the `./autogen.sh` to generate the added or updated aliases. (cf. [DEVELOPMENT][DEVELOPMENT.md])
 
 > IMPORTANT NOTE: To run `autogen.sh`, you need [round][round] and
-> [inkscape][inkscape] command lines that are used for clearning the image
+> [inkscape][inkscape] command lines that are used for learning the image
```
Comment (mingrammer) [inline CONTRIBUTING.md:58]: it should be "cleaning"!

Comment (timgates42) [reply CONTRIBUTING.md]: Thanks - updated

**Review by mingrammer — APPROVED** [review]
LGTM

**Pull request merged**